packages/ra-i18n-i18next/README.md

```diff
@@ -29,7 +30,7 @@ const App = () => {
         options: {
             resources: {
                 en: {
-                    translations: convertRaTranslationsToI18next(englishMessages)
+                    translation: convertRaTranslationsToI18next(englishMessages)
```

Comment:
why?

Reply:
After investigation, you are correct and the doc is wrong. `translation` is the default namespace in i18next (see https://www.i18next.com/overview/configuration-options)